vendor/engines/bulk_email/spec/controllers/bulk_email/bulk_email_controller_spec.rb

```diff
@@ -143,58 +140,53 @@
   describe "POST #deliver" do
     let(:recipients) { FactoryGirl.create_list(:user, 3) }
     let(:custom_message) { "Custom message" }
-    let(:return_path) { nil }
-
-    before(:each) do
-      @action = "deliver"
-      @method = :post
-      @params[:bulk_email_delivery_form] = {
-        custom_subject: custom_subject,
-        custom_message: custom_message,
-        recipient_ids: recipients.map(&:id),
-        search_criteria: {
-          start_date: "12/31/1999",
-          end_date: "1/1/2016",
-          bulk_email: { user_types: ["customers"], products: [1] },
-        }.to_json,
-      }
-      @params[:return_path] = return_path if return_path.present?
-
-      sign_in @admin
-      do_request
+    let(:user) { FactoryGirl.create(:user, :senior_staff, facility: facility) }
+
+    let(:params) do
+      super().merge(bulk_email_delivery_form: {
```

Comment:
It's something I've used a bit before (in 1b6955a931) that I picked up here: http://stackoverflow.com/a/22368792

The parens unfortunately need to be there, otherwise it raises a `RuntimeError`:

`implicit argument passing of super from method defined by define_method() is not supported. Specify all arguments explicitly`